docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
       NODE_ENV: 'development'
       PORT: 8087
       COVERAGE_DIR: htmlcov
-      DATABASE_URL: postgres://postgres@db:5432/postgres
+      DATABASE_URL: ${DATABASE_URL:-postgres://postgres@db:5432/postgres}
```

Comment:
I noticed that this line is still `DATABASE_URL: postgres://postgres@db:5432/postgres` in xpro and open, is there a reason for changing it here?

Reply:
In this project and in xPro, I was continuously getting a Postgres error, likely brought about as some kind of side effect of our Python/Postgres version upgrades. The error was something like "Database is uninitialized and superuser password is not specified.". Allowing for a `.env` override of the `DATABASE_URL` setting allowed me to just specify the password in the connection string itself.